packages/dynamite/dynamite_runtime/lib/src/utils/array_checker.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:built_collection/built_collection.dart';
+
+/// Checks the [input] length against [minItems].
+///
+/// Throws an `Exception` containing the [parameterName] if the `input` is too short.
+void checkMinItems(BuiltList<dynamic>? input, int minItems, String parameterName) {
+  if (input != null && input.length < minItems) {
+    throw FormatException('Parameter "$input" has to be at least $minItems items long');
+  }
+}
+
+/// Checks the [input] length against [maxItems].
+///
+/// Throws an `Exception` containing the [parameterName] if the `input` is too long.
+void checkMaxItems(BuiltList<dynamic>? input, int maxItems, String parameterName) {
+  if (input != null && input.length > maxItems) {
+    throw FormatException('Parameter "$input" has to be at most $maxItems items long');
```

Comment:
This is also wrong on the string check but it should print the `parameterName` instead of the input.